freeipa-client/src/main/java/com/sequenceiq/freeipa/client/FreeIpaCapabilities.java

```diff
@@ -0,0 +1,20 @@
+package com.sequenceiq.freeipa.client;
+
+import com.sequenceiq.freeipa.client.model.Config;
+
+import java.util.Set;
+
+public class FreeIpaCapabilities {
+
+    // users for password setting in FreeIPA Plugin
+    private static final String CDP_USER_ATTRIBUTE = "cdpUserAttr";
+
+    private FreeIpaCapabilities() {
```

Comment:
why have it at all?

Reply:
This class only provides static methods and should not be instantiated, hence private constructor. There are other utility classes in this package do the same. Actually I'll remove the comment so that it's consistent with those classes.